common/lib/xmodule/xmodule/js/js_test.yml

```diff
@@ -40,6 +41,7 @@ lib_paths:
     - common_static/js/vendor/jasmine-imagediff.js
     - common_static/js/vendor/require.js
     - RequireJS-namespace-undefine.js
+    - main_requirejs.js
```

Comment:
@cahrens FYI, we are adding this file in `lib_paths` so that it loads the moment before the actual video files.